### ISSUE TITLE
error：λ:FluentMigrator.Runner.IVersionLoader。[nopcommerce]

### ISSUE THREAD

**Issue by chendelin1982** [Websoft9/docker-library#125]
**Describe the bug**

安装错误：λ:FluentMigrator.Runner.IVersionLoader。

**Comment by chendelin1982** [comment]
8.0 auto plugin is different from 5.7
--default-authentication-plugin=mysql_native_password
```
  mysql:
    image: mysql:$DB_MYSQL_VERSION
    container_name: $APP_NAME-db
    restart: always
    ports:
     - ${DB_MYSQL_PORT}:3306
    command: 
      - --character-set-server=utf8mb4 
      - --collation-server=utf8mb4_unicode_ci
      - --default-authentication-plugin=mysql_native_password

```

but also error

**Comment by chendelin1982** [comment]
Solved: use MySQL 5.7 replace MySQL 8.0
